### DISCUSSION TITLE
Search by socket color.

### DISCUSSION THREAD

**Discussion by Sacson-returnmemynickname** [SnosMe/awakened-poe-trade#833] [Q&A]
Is it possible to add to the search by **socket colors**? To find the **right** colors right away.

**Comment by r2yj2y** [comment]
afaik u can do it in the game by typing the first letters
b-b-b∣r-g-b∣r-r-r∣r-r-b-b-w-w
the divider button ∣ being the separation point that game engine understands

**Reply by Sacson-returnmemynickname** [reply]
**### what are you even talking about?**
